.circleci/config.yml

```diff
@@ -14,5 +14,9 @@ jobs:
           name: update common repo
           command: git submodule update --init
       - run: apt update
+      - run: apt install -y zip
       - run: make release
       - run: make
+      - run:
+          name: make driver
+          command: cd release; cd sht3x; make
```

Comment:
We only build the sht3x driver?

Reply:
`make` builds both, but it's true that only sht3x is rebuilt from the release dir.. we should probably change that